### PR TITLE
Travis Optimisations [WIP]

### PR DESCRIPTION
**Pull request by denisgranha**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -104,8 +105,7 @@ matrix:
     - language: generic
       name: "StableX e2e Tests (Rinkeby)"
       cache:
-        directories:
-          - $HOME/.cache/yarn
+        yarn: true
```
Comment (fleupold) [inline .travis.yml]: should cache pip too?

Comment (denisgranha) [reply .travis.yml]: yep! added it now

**Comment by fleupold** [comment]
🏓 @denisgranha do you want to merge this?

**Comment by fleupold** [comment]
This seems to be outdated.

**Pull request closed without merging**